core/Db.php

```diff
@@ -420,7 +420,7 @@ public static function deleteAllRows($table, $where, $orderBy, $maxRowsPerQuery
      * @param string|array $tables The name of the table to optimize or an array of tables to optimize.
      *                             Table names must be prefixed (see {@link Piwik\Common::prefixTable()}).
      * @param bool $force If true, the `OPTIMIZE TABLE` query will be run even if InnoDB tables are being used.
-     * @return \Zend_Db_Statement
+     * @return bool|\Zend_Db_Statement
```

Comment:
Actually this method now always returns a bool
```suggestion
     * @return bool
```